### PR TITLE
[KYUUBI #2014] Hive Backend Engine - GetCatalog Operation

### PR DESCRIPTION
**Pull request by SteNicholas**
### _Why are the changes needed?_

Hive Backend Engine - GetCatalog Operation.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: externals/kyuubi-hive-sql-engine/src/test/scala/org/apache/kyuubi/engine/hive/operation/HiveOperationSuite.scala [hunk 1]
```diff
@@ -31,6 +31,13 @@ class HiveOperationSuite extends HiveJDBCTestHelper {
     "jdbc:hive2://" + HiveSQLEngine.currentEngine.get.frontendServices.head.connectionUrl + "/;"
   }
 
+  test("get catalogs") {
+    withJdbcStatement() { statement =>
+      val catalogs = statement.getConnection.getMetaData.getCatalogs
+      assert(!catalogs.next())
```
Comment (yaooqinn) [inline externals/kyuubi-hive-sql-engine/src/test/scala/org/apache/kyuubi/engine/hive/operation/HiveOperationSuite.scala:37]: can we also check the value?

**Review by yaooqinn — COMMENTED** [review]
lgtm, only left a minor comment

File: externals/kyuubi-hive-sql-engine/src/test/scala/org/apache/kyuubi/engine/hive/operation/HiveOperationSuite.scala (hunk 1 above)
Comment (SteNicholas) [reply externals/kyuubi-hive-sql-engine/src/test/scala/org/apache/kyuubi/engine/hive/operation/HiveOperationSuite.scala:37]: This line checks whether the `catalogs` is empty list.

**Review by yaooqinn — APPROVED** [review]

**Comment by yaooqinn** [comment]
thanks, merged to master

**Pull request closed without merging**